public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -2334,6 +2334,300 @@ The original idea comes from [Ironsworn](https://ironswornrpg.com)'s vows. `Goal
 
 Special thanks to LynnJones who enhance the readibility, RP for the useful feedback to make it more Charge, and therabidbanana for the progression guideline.
 
+# Party Extra | Extras
+
+An Extra for Charge to add both narrative structure and mechanical benefits to the group of characters represented by the players.
+
+\- By David Haslem
+
+
+## Overview
+
+The group of players is referred to in this extra as a `party`. This group has a separate `party sheet` which tracks `details` of how they are perceived by the world around them, as well as maintaining party-level `danger` and `progress` clocks.
+
+The `party sheet` can represent things such as their “HQ” or “Ship”. It can also be used to represent relationships with important groups or specific NPCs. 
+
+> #### Example
+> 
+> The play group wants to tell the story of a small band of resistance fighters against an evil space empire. They decide they want to start their story with a stolen prototype spaceship and not much else. They have some ideas for characters and they know they want to start on an out-of-the-way space station.
+
+###  Key Concepts and Supporting Details
+
+The `party sheet` tracks four `key concepts`, representing how the party is positioned along four axes. Each `key concept` may have as many as four `supporting details`, which have a `disposition` of positive, negative, or neutral. The combination of these `supporting details` contributes to the overall `key concept` being positive, negative, or neutral.
+
+When `key concepts` or `supporting details` are positive they grant improvements to related rolls the party makes. Negative `key concepts` or `supporting details` cause worse `position` on related rolls.
+
+> #### Example
+>
+> The party decides on the following to start their `party sheet` to represent the story they came up with:
+>
+> Authority of -2, representing the evil empire, with a `key concept` of “Wanted by the Kabraxian Empire”, and two negative `supporting details` of “thorn in the side of the regional governor” and “falsified spacecraft registration”  
+>
+> Assets of +1, with a `key concept` of “scrappy band of fighters ready to fight” and positive `supporting detail` of “recently ‘liberated’ state-of-the-art spacecraft”.
+>
+> Allies of +1, with a `key concept` of “nobody likes the Empire”, a positive `supporting detail` of “Maven, a bar owner friendly to rebels” and a neutral `supporting detail` of “dock workers tired of Empire soldiers pushing them around”.
+>
+> Adversaries of 0, with a `key concept` of “enemies of my enemies are my friends?” and a single neutral `supporting detail` of “Jebralt, the sketchy arms dealer”.
+>
+> All of these `key concepts` are still neutral in `disposition`, but Authority is close to becoming negative. The team will have to keep a low profile for a while if they want to avoid being put into “desperate” positions with the Empire before they’re ready. 
+
```

Comment:
Switching from narrative supporting details and disposition to numerical value is a bit weird and confusing.
Maybe earlier in the disposition description's it would be great to add a sentence about how to convert from narrative to value, just to let the reader be warned about this before discovering it in the example.

Reply:
Yeah, I'm going to remove the numbers completely from the example - I agree it is a bit confusing.